lib/resources.js

```diff
@@ -171,7 +169,8 @@ var parseSearchParams = function (params) {
     isbn: { type: 'string' },
     issn: { type: 'string' },
     lccn: { type: 'string' },
-    oclc: { type: 'string' }
+    oclc: { type: 'string' },
+    merge_checkin_cards: { type: 'bool', default: false }
```

Comment:
Do you think `merge_checkin_cards` is better than [merge_checkin_card_items](https://docs.google.com/document/d/1z5loozi4nH_N6LG5RATJ6Qoyiqi6IDgmt5fP7H5d7So/edit#heading=h.rqv0kh3zd134)? Is this change from spec intentional? I'm willing to go with the change if it's intentional.

Reply:
its not! my bad.